integration-tests/tests/resource-observer/resources.ejs

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Resource observer test</title>
+  <%- renderAgent() %>
+</head>
+<body>
+  <h1>Resource observer test</h1>
+
+  <button id="load-img">Load new Image</button>
+  <button id="load-script">Load new script</button>
+
+  <img id="img-1" src="/integration-tests/assets/splunk-black-white.png" alt="black">
+
+  <script>
+
+    document.getElementById('load-img').addEventListener('click', function() {
+      // random third party img
+      var img = new Image();
+      img.src = 'https://placeimg.com/320/240/any';
```

Comment:
It doesn't seem like a good idea to rely on an external, I'm wondering what the point of doing this is?

Reply:
I use test files for manual testing. This is a nice endpoint for it been using it for years. 
Random sizes and third party.

Reply:
I agree, let's not depend on third parties for basic testing.

Reply:
removed, was just for testing anyways